codec-memcache/src/main/java/io/netty/handler/codec/memcache/DefaultLastMemcacheContent.java

```diff
@@ -33,30 +33,6 @@ public DefaultLastMemcacheContent(ByteBuf content) {
     }
 
     @Override
-    public LastMemcacheContent retain() {
-        super.retain();
-        return this;
-    }
-
-    @Override
-    public LastMemcacheContent retain(int increment) {
-        super.retain(increment);
-        return this;
-    }
-
-    @Override
-    public LastMemcacheContent touch() {
-        super.touch();
-        return this;
-    }
-
-    @Override
-    public LastMemcacheContent touch(Object hint) {
-        super.touch(hint);
-        return this;
-    }
-
-    @Override
```

Comment:
@windie I think these are needed as otherwise touch(...) will not return the correct type. Same is true for others.